builder/dockerfile/copy.go

```diff
@@ -118,9 +118,9 @@ func (o *copier) createCopyInstruction(args []string, cmdName string) (copyInstr
 	if o.platform != nil {
 		pathOS = o.platform.OS
 	}
-	inst.dest = fromSlash(args[last], pathOS)
+	dest := fromSlash(sourcesAndDest.DestPath, pathOS)
```

Comment:
🤦 Ahhhhh.... not sure what I did here; looks like I was planning to construct the `copyInstruction` at the end, instead of contracting a partial one (in the `err` case), but forgot to either complete that, or to roll it back.